src/Details.js

```diff
@@ -1,14 +1,27 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, Redirect } from 'react-router-dom';
+import movies from "./Gallery-get";
```

Comment:
because `Gallery-get` has nothing to do with React (I guess it's just a javascript file returning an array), could you please rename it to the more standard kebap case version, `gallery-get`?

Reply:
Thank you, I have changed it to 'data-get' as it has more information about the movies (synopsis, title, image)

Reply:
Fantastic, sounds great, thanks!